openwisp_radius/tests/test_commands.py

```diff
@@ -174,7 +174,7 @@ def test_delete_old_radiusbatch_users_command(self):
         self.assertEqual(get_user_model().objects.all().count(), 6)
         call_command('delete_old_radiusbatch_users')
         self.assertEqual(get_user_model().objects.all().count(), 3)
-        call_command('delete_old_radiusbatch_users', older_than_months=12)
+        call_command('delete_old_radiusbatch_users', older_than_days=365)
```

Comment:
We need a test that ensures the old behavior continues to work.

Reply:
Sure I can add some more users to test.